src/Stryker.Core/Stryker.Core/Mutants/ICsharpMutantOrchestrator.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Stryker.Core.Mutators;
+
+namespace Stryker.Core.Mutants;
+
+public interface ICsharpMutantOrchestrator
```

Comment:
```suggestion
public interface ICSharpMutantOrchestrator
```

Reply:
The original class is also called `CsharpMutantOrchestrator`..
Do you want me to keep it as is, or do you want me to rename both the interface and the class to `CSharp...`?

Reply:
Would be best to fix the casing